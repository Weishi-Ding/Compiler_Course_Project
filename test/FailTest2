class butterfly {
    string species;
    string color;
    int wingSpan;
}

int main () {
    /*Semanti error: class must have a constructor*/
}