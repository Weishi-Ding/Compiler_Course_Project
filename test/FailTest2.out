Fatal error: exception Dune__exe__Semant.Exception("No constructor has been defined for butterfly")
